### PR TITLE
Use CLI-Toolkit to speed-up FPS

### PR DESCRIPTION
**Pull request by dac514**

### CODE REVIEW

File: src/CliScreen.php [hunk 1]
```diff
@@ -20,47 +21,51 @@ public function __construct()
...
+            $x2 = $c['x'] + $c['w'] - 1;
+            $y2 = $c['y'] + $c['h'];
         }
 
+        CLI\Graphics::line($x1, $y1, $x2, $y2, [$entity->getDrawingCharacter()]);
+
         // Set the cursor to the top left of the window to prevent having
         // a weird cursor flicker follow the last drawn element
-        fwrite(STDOUT, shell_exec('tput cup 0 0'));
+        CLI\Cursor::hide();
```
Comment (nebez) [inline src/CliScreen.php:52]: Does the cursor constantly need hiding after every draw?

**Comment by nebez** [comment]
Nice!
Just FYI it's still dependent on `ncurses`. Check out how it calculates width/height of the screen here: https://github.com/donatj/CLI-Toolkit/blob/master/lib/CLI/Misc.php#L24

File: src/CliScreen.php (hunk 1 above)
Comment (dac514) [inline src/CliScreen.php:52]: For me it had a glitch if I didn't hide it. It could also be moved but hide was an obvious choice.

**Comment by dac514** [comment]
Argh, ok.  At least they cache the response.

**Pull request merged**